crypto/merkletrie/cache.go

```diff
@@ -168,27 +187,37 @@ func (mtc *merkleTrieCache) getNode(nid storedNodeIdentifier) (pnode *node, err
 	if pnode, have = pageNodes[nid]; !have {
 		err = ErrLoadedPageMissingNode
 	} else {
-		mtc.prioritizeNode(nid)
+		mtc.prioritizeNode(nid, mtc.reallocatedPages == nil)
 	}
 	return
 }
 
 // prioritizeNode make sure to adjust the priority of the given node id.
 // nodes are prioritized based on the page the belong to.
-// a new page would be placed on front, and an older page would get moved
+// a new page would be placed on front, and an existing page would get moved
 // to the front.
-func (mtc *merkleTrieCache) prioritizeNode(nid storedNodeIdentifier) {
+func (mtc *merkleTrieCache) prioritizeNode(nid storedNodeIdentifier, frontSide bool) {
```

Comment:
How about splitting this function into two `prioritizeNodeFront` and `prioritizeNodeBack` since they do not share any code now except element finding.
The reason for proposal is to get rid of boolean param from the signature and increase readability.

Reply:
sounds reasonable.